akka-parsing/src/main/scala/akka/macros/LogHelper.scala

```diff
@@ -37,7 +40,7 @@ private[akka] object LogHelper {
       {
         val logHelper = ctx.prefix.splice
         val log = logHelper.log
```

Comment:
`log` can be removed now, only used once below

Reply:
:+1: